### DISCUSSION TITLE
Window Title

### DISCUSSION THREAD

**Discussion by xmha97** [filips123/PWAsForFirefox#167] [Q&A] [answered]
How can I prevent title changes? I want the title to always be the name of the app. (Twitter)
![image](https://user-images.githubusercontent.com/23485114/173378584-5d256556-9a13-41a8-8b4b-0725fcfc64ad.png)
![image](https://user-images.githubusercontent.com/23485114/173379122-e2a43f64-e257-43d5-b6c5-740071b55adf.png)

**Comment by filips123** [comment, marked as answer]
Starting with 2.0.0, you will be able to prevent websites from changing title and icon from the web app settings:

![slika](https://user-images.githubusercontent.com/16626308/182028039-11e75ce1-2746-4584-8acd-5f4fb0619d86.png)
